package.json

```diff
@@ -6,7 +6,10 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "prettier": "prettier \"**/*.+(js)\"",
```

Comment:
Sorry, I just did a mistake in the issue description. I just realised, it shouldn't be just .js file

can you please update the prettier command to

`"prettier": "prettier \"**/*.+(js|json|ts|jsx|tsx)\""`